io/net/pktgen.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Sandeep K <sandeep@linux.vnet.ibm.com>
+#
+# Based on code by Lucas Meneghel Rodrigues
+# https://github.com/autotest/autotest-client-tests/tree/master/pktgen
+
+import os
+import re
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+
+
+class Pktgen(Test):
+
+    '''
+    Pktgen is software based traffic generator powered by
+    DPDK fast packet processing framework
+    '''
+
+    def setUp(self):
+        self.eth = self.params.get("eth", default="eth0")
+        self.count = self.params.get("count", default="50000")
+        self.clone_skb = self.params.get("clone_skb", default="1")
+        self.dst_ip = self.params.get("dst_ip", default="")
+        self.dst_mac = self.params.get("dst_mac", default="")
+        self.resultsdir = self.params.get("resultsdir", default="/root/")
+
+        if not os.path.exists('/proc/net/pktgen'):
+            process.system("modprobe pktgen", shell=True)
```

Comment:
can you please use ignore_status=True parameter in process.sytem(). if command fail script will error out